### ISSUE TITLE
Integrating this python chatbot on a PHP website

### ISSUE THREAD

**Issue by Siddikulus** [gunthercox/ChatterBot#2055]
Hello,

The bot I've built works fine and I want to integrate it on my PHP website. 

What I'm doing as of now is requesting the user a question through PHP, passing this question as a parameter to the python chatbot script, getting a response and displaying on the website.

Even though it works, there isn't much flexibility with what the bot can do. For example, If I want to display an element from my website, let's say the current time on the website (I know about the time logic adapter, just taking this as an example) I can't do that with the bot.
 
Is there a way to completely integrate my chatbot with my PHP website like it is in Django?

**Comment by AnAnalogGuy** [comment]
There is an example with Flask. 
https://github.com/chamkank/flask-chatterbot

**Comment by gunthercox** [comment]
Yeah, I'd say the best way to go about something like that would be to build a Django or Flask app for your ChatterBot application and access it from the PHP site by implementing an API. [Django REST Framework](https://www.django-rest-framework.org/tutorial/quickstart/) is a great example of one possible way to go about implementing an API.

***

Closing this issue off since its old.
